vulture_os/authentication/user_portal/views.py

```diff
@@ -125,10 +126,14 @@ def render_form(profile, **kwargs):
         try:
             if api:
                 repo_attrs = request.JSON.get('repo_attributes', [])
-                assert isinstance(repo_attrs, list), "Repo attributes field must be a dict."
+                assert isinstance(repo_attrs, list), "Repo attributes field must be a list."
             else:
                 repo_attrs = json_loads(request.POST.get('repo_attributes', "[]"))
         except Exception as e:
+            if api:
+                return JsonResponse({
+                    "error": "".join(format_exception(*exc_info()))
+                })
```

Comment:
status code 500 ?

(+ 2 others at 151 and 198)

Reply:
Status code 400. Fixed with [843e982](https://github.com/VultureProject/vulture-gui/pull/86/commits/834e9829c80ae7fb68268f3cabb4fc613a3ec1dc)